Assets/Scripts/Actors/ActorPhysics.cs

```diff
@@ -22,6 +22,10 @@ public sealed class ActorPhysics : ActorComponent
 	PhysicsStateType _currentStateType = PhysicsStateType.Jumping;
 	PhysicsState _currentState = new DefaultState();
 	Dictionary<PhysicsStateType, PhysicsState> _stateMap = new Dictionary<PhysicsStateType, PhysicsState>();
+
+	public delegate void StateCallback( PhysicsStateType physicsStateType );
```

Comment:
consider removing exit state callback if no one's using it.
Also consider making it one callback like ChangeStateCallBack( PhysicsStateType oldState, PhysicsStateType newState )